mne/viz/misc.py

```diff
@@ -309,7 +309,6 @@ def _plot_mri_contours(mri_fname, surfaces, src, orientation='coronal',
     """Plot BEM contours on anatomical slices."""
     import matplotlib.pyplot as plt
     from matplotlib import patheffects
-    from .._freesurfer import _mri_orientation, _read_mri_info
```

Comment:
This is better, internal MNE imports should be nested only when required by import cycle problems